### ISSUE TITLE
Right to left text/translation support

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#24954]
**Is your feature request related to a problem? Please describe.**
We support only left to right text on translations, but there are many people that need right to left text translations. I'm clearly not an expert on languages but a customer mentioned this

**Describe the solution you'd like**
Support languages that are written from right to left

**Describe alternatives you've considered**
NA

**How important is this feature to you?**
Mentioned by a customer

**Additional context**
NA

**Comment by AsimNet** [comment]
👍

**Comment by okaeiz** [comment]
Is this issue being considered?
The RTL support for Persian language is highly needed. The strings are translated but the alignment is LTR.

**Comment by elkhawajah** [comment]
If Metabase team can provide a brief requirement/guidance for what need to be added/updated to support RTL, we can support as a community and I am welling to invest some with my developers to do initial version.

**Comment by paoliniluis** [comment]
@elkhawajah i would suggest you set up the development environment and try checking out our development documentation

**Comment by okaeiz** [comment]
Any updates?

**Comment by seidnerj** [comment]
Better yet, could someone guide us where is the relevant code so that we can work on adding support? 🙏

**Comment by elkhawajah** [comment]
I see this PR has it https://github.com/metabase/metabase/pull/40031 but not sure if it includes the end user dashboard rendering as well, admin side is least important compared to the rendering of the dashboards in RTL

**Comment by okaeiz** [comment]
> I see this PR has it [#40031](https://github.com/metabase/metabase/pull/40031) but not sure if it includes the end user dashboard rendering as well, admin side is least important compared to the rendering of the dashboards in RTL

Agreed. Although the admin panel being right to left is useful and great for RTL users, but the end user dashboards and shared questions are more important here since they are exposed to a wider range of people (the ones who just intend to see the reports and do not care how they have been created).
